pkg/cmd/extension/extension.go

```diff
@@ -49,8 +49,7 @@ func (e *Extension) IsPinned() bool {
 }
 
 func (e *Extension) UpdateAvailable() bool {
-	if e.isPinned ||
-		e.isLocal ||
+	if e.isLocal ||
```

Comment:
removing `e.isPinned` here as it is checked just before this function is called in https://github.com/cli/cli/blob/ba1df3e6575a099fe89f1835f7abf2c7ae4b38dc/pkg/cmd/extension/manager.go#L578

Reply:
This may be a bit of scope creep, but since we are touching this method, it would be nice to add some tests for the new behavior. I know there are not tests for `UpdateAvailable()` yet, but seems like it would be fairly easy to add.

Reply:
@samcoe I added unit tests in https://github.com/cli/cli/pull/6513/commits/8a320b63d7beed1a10e3c2bf6cf60ac15debca9a not sure if that's what you had in mind?